build.gradle

```diff
@@ -173,12 +173,10 @@ def jarName = 'iexec-worker.jar'
 def jarPath = buildDir + jarName
 
 task copyJar(type: Copy) {
```

Comment:
```suggestion
task copyBootJar(type: Copy) {
```
?

Reply:
Honestly, I don't know if I want to do this.
* I would prefer to keep it minimal because this is the only module with this implementation.
* It would be more interesting to decide on one implementation for all modules

But if you want a task rename, I don't really have a problem with it

Reply:
The name `copyBootJar` seems clearer to me.

Reply:
ok, I'm on it